receiver/windowsperfcountersreceiver/internal/pdh/pdh_test.go

```diff
@@ -80,12 +80,50 @@ func TestPerfCounter_Close(t *testing.T) {
 }
 
 func TestPerfCounter_ScrapeData(t *testing.T) {
-	pc, err := NewPerfCounter(`\Memory\Committed Bytes`, false)
-	require.NoError(t, err)
+	type testCase struct {
+		name           string
+		path           string
+		assertExpected func(t *testing.T, data []win_perf_counters.CounterValue)
+	}
 
-	performanceCounters, err := pc.ScrapeData()
-	require.NoError(t, err, "Failed to scrape data: %v", err)
+	testCases := []testCase{
+		{
+			name: "no instances",
+			path: `\Memory\Committed Bytes`,
```

Comment:
I don't know this exporter, so, just to clarify: what's the instance name in the test cases? Is it whatever is inside parenthesis?

Reply:
Receiver you mean?

But yes you are correct; if relevant, the instance name is defined inside the brackets. This is how you specify perf counter paths in Windows using PDH, see https://docs.microsoft.com/en-us/windows/win32/perfctrs/specifying-a-counter-path

Some perf counters do not have instances associated with them, e.g. anything under the `Memory` object, in which case there will be no parenthesis.

Reply:
Receiver, yes, sorry :-) Thanks for the clarification, sounds good!